src/table/components/TableWrapper.jsx

```diff
@@ -124,10 +130,10 @@ export default function TableWrapper(props) {
         data-testid="table-container"
       >
         <Table stickyHeader aria-label={tableAriaLabel}>
-          <TableHeadWrapper {...props}>
-            {totalsPosition === 'top' ? <TableTotals {...props} /> : undefined}
-          </TableHeadWrapper>
-          <TableBodyWrapper {...props} setShouldRefocus={setShouldRefocus} tableWrapperRef={tableWrapperRef} />
+          <TableHeadWrapper {...props} updateCellHeight={updateCellHeight} />
+          <TableBodyWrapper {...props} setShouldRefocus={setShouldRefocus} tableWrapperRef={tableWrapperRef}>
+            {totalsPosition === 'top' ? <TableTotals {...props} height={height} /> : undefined}
```

Comment:
This is where we want to pass it now. Just move the position logic inside the table body